src/entrypoints/app.ts

```diff
@@ -1,11 +1,10 @@
-import {
-  setPassiveTouchGestures,
-  setCancelSyntheticClickEvents,
-} from "@polymer/polymer/lib/utils/settings";
 import "@webcomponents/scoped-custom-element-registry/scoped-custom-element-registry.min";
 import "../layouts/home-assistant";
 
 import("../resources/ha-style");
-
-setPassiveTouchGestures(true);
-setCancelSyntheticClickEvents(false);
+import("@polymer/polymer/lib/utils/settings").then(
+  ({ setCancelSyntheticClickEvents, setPassiveTouchGestures }) => {
+    setCancelSyntheticClickEvents(false);
+    setPassiveTouchGestures(true);
```

Comment:
Why is this being set anyway when `preventDefault()` is called all over the place?

Reply:
Those are mostly for click action and other events, not for touch events